themes/bootstrap3/js/common.js

```diff
@@ -139,13 +165,17 @@ var VuFind = (function VuFind() {
     addTranslations: addTranslations,
     init: init,
     emit: emit,
+    getCspNonce: getCspNonce,
```

Comment:
Is there a compelling reason to have a public getter? Does making the nonce available in this way increase security risks? Do these JS tools make it potentially easier for untrusted code to get executed? (I obviously can't think of any reason why somebody would bother to write a specialized, VuFind-targeted vector to inject malicious code... but regardless of that, I'm just trying to think through all the possibilities).

Reply:
I wanted to make it available for any situation where the basic updateCspNonce doesn't do the job. While I'm not currently aware of such case, this should pose no security issue. If you can inject js to call getCspNonce, it's already game over, and you could just as well read it from a script tag in the DOM.